test/integration/client-side-encryption/client_side_encryption.prose.14.decryption_events.test.ts

```diff
@@ -218,6 +228,13 @@ describe('14. Decryption Events', metadata, function () {
       const collection = encryptedClient.db('db').collection('decryption_events');
       await collection.insertOne({ encrypted: malformedCiphertext });
 
+      /// Verify the malformedCiphertext was inserted with a plain client
+      const docCount = setupClient.db('db').collection('decryption_events').countDocuments({});
```

Comment:
I think what we want here is not to count the documents, but rather to find all documents and assert that the malformed document is present.  If it is, and the aggregation fails, that indicates a bug in our FLE code. That's what I want to determine with this change

Reply:
Maybe we should add write concern majority to the insertion just in case too